cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from cinema.views import *
```

Comment:
It's a bad practice to import everything with wildcard (*). Also, add blank line between third-party and custom libraries.